core/scripts/cli/sponsor/marketUtils.js

```diff
@@ -74,6 +79,44 @@ const getMarketSummary = async (web3, artifacts) => {
   return markets;
 };
 
+async function getCollateralizationRatio(web3, empAddress, collateral, tokens) {
+  const { toBN, toWei } = web3.utils;
+  let priceFeed;
+  try {
+    priceFeed = await createReferencePriceFeedForEmp(
+      winston.createLogger({ silent: true }),
```

Comment:
It would be nice to not have to feed in a silent logger for these kinds of non-bot usage 🤔 . This ends up bloating this CLI by requiring this package, even thought its not used.